tools/log_rotate/install_script.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Create directory, which keeps all logrotate-config to rotate hourly.
+mkdir /etc/logrotate.hourly.d
+
+cat << EOF | tee /etc/logrotate.hourly.conf
```

Comment:
I think it would be really helpful to add one line comment about the logic here. Same for next cat block.

Reply:
Added.